config.py

```diff
@@ -45,6 +45,10 @@ class Config(object):
                                                         RAS_SECURE_MESSAGE_SERVICE_HOST,
                                                         RAS_SECURE_MESSAGE_SERVICE_PORT)
 
+    PARTY_SERVICE_URL = os.getenv('PARTY_SERVICE_URL')
+    PARTY_SERVICE_USERNAME = os.getenv('PARTY_SERVICE_USERNAME')
+    PARTY_SERVICE_PASSWORD = os.getenv('PARTY_SERVICE_PASSWORD')
```

Comment:
I believe we're going for a config pattern where we remove `service` in the naming

Reply:
Where is this documented?

Reply:
This needs discussed with the tech leads if some people are doing one thing and others doing something else for consistency purposes. In this case it doesn't make sense to me to remove the service as PARTY refers to something entirely different in regards to the applications as we have a respondent party or business party then the party service that handles this.